deployment/kubernetes/objects/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: objects
+description: A Helm chart for Objects API
+
+type: application
+version: 0.1.0
```

Comment:
Should this match our Docker container version? If so, should it be added to bumpversion config?

Reply:
Afaik this is a helm chart version and it should be incremented when the helm chart itself is changed
Docker container is deployed with "latest" tag now, which is specified in `value.yaml` file